projects/connectedhomeip/project.yaml

```diff
@@ -0,0 +1,14 @@
+homepage: "https://buildwithmatter.com/"
+language: c++
+primary_contact: "andreilitvin@google.com"
+auto_ccs:
+  - "mboyington@google.com"
+sanitizers:
+  - address
+  - undefined
+  - memory
+fuzzing_engines:
+  - libfuzzer
+  - honggfuzz
+  - centipede
```

Comment:
no afl?

Reply:
I had it working locally, but I would need to submit another PR to the target project before enabling it here.